pkg/investigations/ccam/ccam.go

```diff
@@ -70,49 +49,7 @@ func Evaluate(cluster *v1.Cluster, awsError error, ocmClient ocm.Client, pdClien
 		return pdClient.SilenceAlertWithNote(fmt.Sprintf("Skipped adding limited support reason '%s': cluster is already uninstalling.", ccamLimitedSupport.Summary))
 	default:
 		// Anything else is an unknown state to us and/or requires investigation.
-		// E.g. we land here if we run into a CPD alert where credentials were removed (installing state)
+		// E.g. we land here if we run into a CPD alert where credentials were removed (installing state) and don't want to put it in LS yet.
 		return pdClient.EscalateAlertWithNote(fmt.Sprintf("Cluster has invalid cloud credentials (support role/policy is missing) and the cluster is in state '%s'. Please investigate.", cluster.State()))
 	}
 }
-
-// RemoveLimitedSupport will remove any CCAM limited support reason from the cluster,
-// if it fails to do so, it will try to alert primary
-// Run this after cloud credentials are confirmed
-func RemoveLimitedSupport(cluster *v1.Cluster, ocmClient ocm.Client, pdClient pagerduty.Client, alertType string) error {
```

Comment:
question: i was under the impression, we keep the CCAM LS logic as is for now. 
With this change we will continue to put clusters into LS for ccam, but do not remove it anymore.

Reply:
I think this might be our current best option, see above comment. Possibly removing a LS reason that is not really related to an alert when an alert resolves is kind of odd behaviour.